packages/portalnetwork/src/networks/contentLookup.ts

```diff
@@ -193,6 +193,7 @@ export class ContentLookup {
         this.logger(`received ${res.enrs.length} ENRs for closer nodes`)
         for (const enr of res.enrs) {
           const decodedEnr = ENR.decode(enr as Uint8Array)
+          this.network.portal.updateENRCache([decodedEnr])
```

Comment:
Should we actually update it here or should we wait until we get a PONG from the node to know its a live node?